documentation/release_notes.rst

```diff
@@ -8,6 +8,26 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
 and provides high-productivity APIs aimed to minimize programming efforts of C++ developers
 creating efficient heterogeneous applications.
 
+New in 2022.0.0
+===============
+
+New Features
+------------
+- Added the functionality from ``<complex>`` and more APIs from ``<cmath>`` and ``<limits>`` standard headers to `Tested Standard C++ API`_.
+- Improved performance of ``sort`` and ``stable_sort``  algorithms on GPU devices when using Radix sort [#fnote1]_.
+
+Fixed Issues
+------------
+- Fixed an error in ``oneapi::dpl::experimental::ranges::guard_view`` and ``oneapi::dpl::experimental::ranges::zip_view`` when using ``operator[]`` with an index exceeding the limits of a 32 bit integer type.
```

Comment:
Let's avoid such long lines; some might want to read this as a text file.
Besides this line, we have long lines in a few previous notes; it would be good to split those as well.

Reply:
I've shorted this line and few others. If more shortening is needed, I'll add it in separate PR